test/extensions/filters/http/aws_lambda/aws_lambda_filter_test.cc

```diff
@@ -69,6 +69,16 @@ TEST_F(AwsLambdaFilterTest, DecodingHeaderStopIteration) {
   EXPECT_EQ(Http::FilterHeadersStatus::StopIteration, result);
 }
 
+/**
+* Requests should have their host header altered, resulting in a host header of "lambda"
+*/
```

Comment:
you might have to fix the formatting error complained by the CI job somewhere here.


Meanwhile I would suggest putting this change behind a reloadable_features which is true by default.

Reply:
Where is reloadable_features located to put it behind that?

Reply:
Not sure what's up with the formatter.  My second and third line in my comment weren't properly indented which is fixed, now the formatter is telling me to unindent them after it told me to indent them....

Reply:
https://github.com/envoyproxy/envoy/blob/1ef7992acc2259b208b1c8c5ea84dcd33f2b072d/source/common/runtime/runtime_features.cc you can find the reloadable_features in this file. You can refer some of the recent ones to see how they are used. Also might need to test true and false scenarios in the unit test.